app/models/network/node.rb

```diff
@@ -1,5 +1,7 @@
 module Network
   class Node < Turbine::Node
+    include BusinessCaseCosts
```

Comment:
This stuff doesn't belong here. :smile:

The `Network` namespace is for calculating load. Anything relating to costs should be part of the `Market` module.